plugins/modules/azure_rm_keyvaultsecret_info.py

```diff
@@ -353,7 +354,7 @@ def list_secrets(self):
 
             if response:
                 for item in response:
-                    item = self.get_secret(item._id.split('/')[-1])
+                    item = self.get_secret(item._id.split('/')[-1]) if item._attributes.enabled else None
```

Comment:
Can you add a comment above to mention the API will error out if the secret is disabled.